src/Oci8/Query/Grammars/OracleGrammar.php

```diff
@@ -597,6 +597,25 @@ protected function whereInRaw(Builder $query, $where)
         return '0 = 1';
     }
 
+    /**
+     * Compile a "where fulltext" clause.
+     *
+     * @param  \Illuminate\Database\Query\Builder  $query
+     * @param  array  $where
+     * @return string
+     */
+    public function whereFullText(Builder $query, $where)
+    {
+        $columns = collect($where['columns'])
+            ->map(fn ($column) => $this->wrap($column))
+            ->implode(' || ');
```

Comment:
I tried using this on multiple fulltext indexed columns and it doesn't seem to work. Is this intended or is there any use for this?

Reply:
Factually... Fixed this logic.